### DISCUSSION TITLE
Print floating value is not working for me

### DISCUSSION THREAD

**Discussion by Bharris100** [polarfire-soc/polarfire-soc-documentation#300] [How to] [answered]
Hi,
Platform: windows
UART0 terminal: extraPuTTy
Hart1 is using UART0

Problem:
I am trying to print a floating point value to UART0 using the following statements:
 sprintf(info_string,"%.2f C\r\n", tecTemp);
 MSS_UART_polled_tx_string(&g_mss_uart0_lo, info_string);
The output is " C". It SB value.value I believe.
I can print a hex type, a decimal type, but not a float type.
Your help is appreciated,
Bharris

**Comment by eppidei** [comment, marked as answer]
![float](https://user-images.githubusercontent.com/41241280/224391188-1b93d6ad-1131-42ce-a335-a4f413024d26.PNG)

**Comment by Bharris100** [comment]
This was the issue.  Thank You!
